### PR TITLE
8287897: Augment src/jdk.internal.le/share/legal/jline.md with information on 4th party dependencies

### PR DESCRIPTION
**Pull request by lahodaj**
---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.java.net/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8287897](https://bugs.openjdk.org/browse/JDK-8287897): Augment src/jdk.internal.le/share/legal/jline.md with information on 4th party dependencies

### Reviewers
 * [Vicente Romero](https://openjdk.java.net/census#vromero) (@vicente-romero-oracle - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/9065/head:pull/9065` \
`$ git checkout pull/9065`

Update a local copy of the PR: \
`$ git checkout pull/9065` \
`$ git pull https://git.openjdk.java.net/jdk pull/9065/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 9065`

View PR using the GUI difftool: \
`$ git pr show -t 9065`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/9065.diff">https://git.openjdk.java.net/jdk/pull/9065.diff</a>

</details>

### CODE REVIEW

File: src/jdk.internal.le/share/legal/jline.md [hunk 1]
```diff
@@ -38,4 +38,252 @@ LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
...
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY,   FITNESS   FOR   A  PARTICULAR   PURPOSE   AND
+NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+These terms are identical to those of the MIT License, also called the X License or the X11 License, which is a simple, permissive non-copyleft free software license. It is deemed compatible with virtually all types of licenses, commercial or otherwise. In particular, the Free Software Foundation has declared it compatible with GNU GPL. It is also known to be approved by the Apache Software Foundation as compatible with Apache Software License.
```
Comment (vicente-romero-oracle) [inline src/jdk.internal.le/share/legal/jline.md:287]: there seem to be some indentation issues here

File: src/jdk.internal.le/share/legal/jline.md [hunk 2]
```diff
@@ -38,4 +38,252 @@ LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
...
+     To apply the Apache License to your work, attach the following
+     boilerplate notice, with the fields enclosed by brackets "[]"
+     replaced with your own identifying information. (Don't include
+     the brackets!)  The text should be enclosed in the appropriate
+     comment syntax for the file format. We also recommend that a
+     file or class name and description of purpose be included on the
+     same "printed page" as the copyright notice for easier
+     identification within third-party archives.
+
+  Copyright [yyyy] [name of copyright owner]
```
Comment (vicente-romero-oracle) [inline src/jdk.internal.le/share/legal/jline.md:237]: we need to update the template here?

File: src/jdk.internal.le/share/legal/jline.md (hunk 1 above)
Comment (lahodaj) [reply src/jdk.internal.le/share/legal/jline.md]: Done.

File: src/jdk.internal.le/share/legal/jline.md (hunk 2 above)
Comment (lahodaj) [reply src/jdk.internal.le/share/legal/jline.md:237]: I believe this text is a template, that can be filled by others, so this should not be modified.

File: src/jdk.internal.le/share/legal/jline.md [hunk 3]
```diff
@@ -38,4 +38,257 @@ LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
...
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+=============
+juniversalchardet
+
+The library is subject to the Mozilla Public License Version 1.1.
+
+Alternatively, the library may be used under the terms of either the GNU General Public License Version 2 or later, or the GNU Lesser General Public License 2.1 or later.
```
Comment (vicente-romero-oracle) [inline src/jdk.internal.le/share/legal/jline.md:256]: should we fix the indentation of this line?

**Review by vicente-romero-oracle — APPROVED** [review]
just added a nit additional comment, there is no need for another iteration

**Pull request closed without merging**